### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by immber**
## What does this PR do?
Updates documentaton

## How do I test this PR?

verify netlify preivew

### CODE REVIEW

**Review by kgardnr — APPROVED** [review]
Looks great to me! Just letting @wyattjoh give final OK 💯

File: docs/source/integrating/asset-management.md [hunk 1]
```diff
@@ -3,28 +3,85 @@ title: Asset Management
...
+## Triggering the Comments Section (client side, i.e. Your site)
+
+When the embed script is triggered on your page load several things are initiated inside Talk, including fetching all comments for the specified article, establishing websocket connections for this user, and checking user’s session for SSO/authentication. 
+
+Instead of greedily triggering the embed to render on _EVERY PAGE LOAD_, we highly recommend implementing a _“lazy”_ rendering strategy to only render the comments section if a user wants to interact with it. This will greatly improve your initial page load performance, and will be critical to managing server resources if you’re running Talk on a heavy-traffic production site. 
+
+We recommend using one of these _“lazy”_ loading strategies:
+
+#### Scroll to Comments Section
+Wait for user to scroll to the comment section before triggering the embed to render.
```
Comment (wyattjoh) [inline docs/source/integrating/asset-management.md:29]: Add a note that this requires at least v4.6.8

File: docs/source/integrating/asset-management.md [hunk 2]
```diff
@@ -3,28 +3,85 @@ title: Asset Management
...
+    $.getScript('<TALK_ROOT_URL>/static/embed.js', function() {
+      Coral.Talk.render(document.getElementById('coral_talk_stream'), {
+        talk: '<TALK_ROOT_URL>',
+      });
+    });
+  });
+</script>
+```
+
+## Creating Assets in Talk (server side, i.e. What you need to send to Talk)
```
Comment (wyattjoh) [inline docs/source/integrating/asset-management.md:65]: I'd also suggest moving all this content to a new page (as the embed is unrelated to asset management).

**Review by wyattjoh — CHANGES_REQUESTED** [review]
Some small comments!

File: docs/source/integrating/asset-management.md (hunk 1 above)
Comment (immber) [reply docs/source/integrating/asset-management.md:29]: will add a note here

File: docs/source/integrating/asset-management.md (hunk 2 above)
Comment (immber) [reply docs/source/integrating/asset-management.md:65]: Going to rename this page so that it's clearer

**Review by wyattjoh — APPROVED** [review]

**Pull request merged**